crates/rerun/src/web_viewer.rs

```diff
@@ -17,19 +17,21 @@ struct WebViewerSink {
 }
 
 impl WebViewerSink {
+    /// A `bind_ip` of `"0.0.0.0"` is a good default.
     pub fn new(
         open_browser: bool,
+        bind_ip: &str,
         web_port: WebViewerServerPort,
         ws_port: RerunServerPort,
     ) -> anyhow::Result<Self> {
         let (rerun_tx, rerun_rx) = re_smart_channel::smart_channel(re_smart_channel::Source::Sdk);
 
-        let rerun_server = RerunServerHandle::new(rerun_rx, ws_port)?;
-        let webviewer_server = WebViewerServerHandle::new(web_port)?;
+        let rerun_server = RerunServerHandle::new(rerun_rx, bind_ip.to_owned(), ws_port)?;
+        let webviewer_server = WebViewerServerHandle::new(bind_ip, web_port)?;
 
         let web_port = webviewer_server.port();
         let server_url = rerun_server.server_url();
-        let viewer_url = format!("http://127.0.0.1:{web_port}?url={server_url}");
+        let viewer_url = format!("http://{bind_ip}:{web_port}?url={server_url}");
```

Comment:
Interesting. This looks problematic to me and I was surprised that it worked when using 0.0.0.0.

See: https://unix.stackexchange.com/questions/419880/connecting-to-ip-0-0-0-0-succeeds-how-why

Either way it means even if 0.0.0.0 works for localhost the url you generate will still be wrong: 0.0.0.0 most definitely won't work if you're trying to access from another computer.

I think the proper thing to do here, similar to how we handle the ephemeral port, is to expose, e.g.:
```
bound_ip = webviewer_server.ip()
```
and actually resolve that ip post bind.

Reply:
`127.0.0.1` doesn't work from another computer either.

I've made a commit where I ask the running server for its `local_addr` (its where we get the resolved port from). On my machine that still reports `0.0.0.0` which again, works locally but not from another computer (just like `127.0.0.1`).

Reply:
Yeah, I was just looking at your pushed change and noting the same thing.